sass/components/_tools.scss

```diff
@@ -0,0 +1,50 @@
+.tools {
+  padding: 48px 0 90px;
+  background: url(../assets/img/tools/background.svg) center bottom no-repeat;
+  @include media-small-desktop {
+    padding: 60px 20px 90px;
+  }
+  @include media-mobile-small {
+    padding: 30px 10px 45px;
+  }
+}
+
+.tools__wrapper {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  max-width: 1095px;
+}
+
+.tools__title {
+  max-width: 539px;
+  text-align: center;
+}
+
+.tool {
+  @include media-mobile-big {
```

Comment:
Not fully correct naming
It would be better to use 'media-mobile-large' here
By the way, we will discuss this a little bit later, when we will add mobile version